USING.md

```diff
@@ -73,7 +93,8 @@ Older versions of JTS are available on Maven Central.
     <version>1.14.0</version>
 </dependency>
 ```
-### 1.13
+
+JTS 1.13:
```

Comment:
Do we want this to be bold?  If so, we should keep the hashes, no?  (I don't do much with MD, but I think that's right.)

Reply:
hashes are for headings, (bold / strong / emphasis uses double asterix), semantically you probably need 4-hashes `####` to denote sub-heading (definition-lists are a total pain in GH-markdown)